src/mochiweb_http.erl

```diff
@@ -48,9 +48,11 @@ parse_options(Options) ->
     mochilists:set_defaults(?DEFAULTS, Options1).
 
 stop() ->
+    mochiweb_clock:stop(),
```

Comment:
`mochiweb_http` should only start, not stop the clock. There can be many `mochiweb_http` instances, but there should only be one clock. If the lifecycle is going to be managed, it would need to be done with reference counting or something.